### PR TITLE
updated IdP note

### PR DESCRIPTION
**Pull request by jerae-duffin**
Updated IdP note and configuration for Azure AD.

### CODE REVIEW

File: single-sign-on/index.md [hunk 1]
```diff
@@ -36,9 +36,15 @@ In addition, all email addresses should be added to your IdP.
 
 To configure SSO, log into [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} to obtain the **ACS URL** and **Entity IDs** to complete the IdP server configuration process. You can only configure SSO with a single IdP.  When this is complete, log back into [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} and complete the SSO enablement process.
 
-> **Note:**
+> **Important**
 >
-> IdP initiated connections are not supported at this time
+> If your IdP setup requires an Entity ID and the ACS URL, you must select the
+> SAML tab in the Authentication Method section. For example, if your Azure AD
```
Comment (usha-mandya) [inline single-sign-on/index.md:42]:
```suggestion
> **SAML** tab in the **Authentication Method** section. For example, if your Azure AD
```

File: single-sign-on/index.md [hunk 2]
```diff
@@ -36,9 +36,15 @@ In addition, all email addresses should be added to your IdP.
...
+> setup uses SAML configuration within Azure AD, you must select SAML. If you
```
Comment (usha-mandya) [inline single-sign-on/index.md:43]:
```suggestion
> setup uses SAML configuration within Azure AD, you must select **SAML**. If you
```

File: single-sign-on/index.md [hunk 3]
```diff
@@ -36,9 +36,15 @@ In addition, all email addresses should be added to your IdP.
...
+> are [configuring Open ID Connect with Azure AD](https://docs.microsoft.com/en-us/powerapps/maker/portals/configure/configure-openid-settings){:
```
Comment (usha-mandya) [inline single-sign-on/index.md:44]: We don't need to the CTA ID for the URL as we are not redirecting users to a Docker website

File: single-sign-on/index.md [hunk 4]
```diff
@@ -36,9 +36,15 @@ In addition, all email addresses should be added to your IdP.
...
+ target="_blank" rel="noopener" class="_" id="dkr_docs_cta"}, select the Azure AD as the
```
Comment (usha-mandya) [inline single-sign-on/index.md:45]:
```suggestion
 target="_blank" rel="noopener" class="_" id="dkr_docs_cta"}, select the **Azure AD** as the
```

File: single-sign-on/index.md [hunk 5]
```diff
@@ -71,7 +77,7 @@ The following video walks you through the process of configuring SSO.
 
 7. Proceed to **add your domain** before you test and enforce SSO.
 
-### Azure AD IdP configuration
+### Azure AD IdP configuration with Open ID Connect
```
Comment (usha-mandya) [inline single-sign-on/index.md:81]: Some users may land directly on this section. Can we also add the note here to convey that users must follow these instructions only when configuring Open ID Connect with Azure AD?

File: single-sign-on/index.md (hunk 1 above)
Comment (jerae-duffin) [reply single-sign-on/index.md:42]: Updated

File: single-sign-on/index.md (hunk 3 above)
Comment (jerae-duffin) [reply single-sign-on/index.md:44]: Thanks Usha! Removing it

File: single-sign-on/index.md (hunk 5 above)
Comment (jerae-duffin) [reply single-sign-on/index.md:82]: Updated

File: single-sign-on/index.md (hunk 4 above)
Comment (jerae-duffin) [reply single-sign-on/index.md]: Updated

File: single-sign-on/index.md (hunk 1 above)
Comment (jerae-duffin) [reply single-sign-on/index.md]: Updated

**Review by technicallyjosh — APPROVED** [review]
LGTM. It may be worth noting that this is not a full OIDC integration with any customization. This is a very basic integration at it's core.

File: single-sign-on/index.md [hunk 6]
```diff
@@ -71,7 +77,12 @@ The following video walks you through the process of configuring SSO.
...
+
+>**Note:**
+>
+> This section is for users who only want to configure Open ID Connect with
+> Azure AD. This connection is not a full OpenID Connect with customization.
```
Comment (technicallyjosh) [inline single-sign-on/index.md:85]:
```suggestion
> Azure AD. This connection is a basic OIDC connection and there are no special customizations available when using it.
```

**Review by technicallyjosh — APPROVED** [review]
Thanks 😄  Just added a suggestion.

File: single-sign-on/index.md (hunk 6 above)
Comment (jerae-duffin) [reply single-sign-on/index.md]: Thanks Josh, updated!

**Pull request merged**